src/components/TodoForm.js

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import CssBaseline from "@mui/material/CssBaseline";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Grid from "@mui/material/Grid";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { useState } from "react";
+import { FormControl } from "@mui/material";
+import CheckboxList from "./CheckboxList";
+
+function TodoForm() {
+  const [text, setText] = useState("");
+  const [taskId, setTaskId] = useState(0);
+  const [tasks, setTasks] = useState([]);
+
+  const addTask = (text) => {
+    setTaskId(taskId + 1);
+    const newtask = {
+      id: taskId,
+      title: text,
+      iscompleted: false,
+    };
+    setTasks([...tasks, newtask]);
```

Comment:
well done here! also the name of the function is pretty good because the name describe what the function does.

Reply:
thank you!